plugin/trino-http-event-listener/src/main/java/io/trino/plugin/httpquery/HttpEventListenerConfig.java

```diff
@@ -34,7 +41,22 @@ public class HttpEventListenerConfig
     private final EnumSet<HttpEventListenerEventType> loggedEvents = EnumSet.noneOf(HttpEventListenerEventType.class);
     private String ingestUri;
     private HttpEventListenerHttpMethod httpMethod = HttpEventListenerHttpMethod.POST;
-    private Map<String, String> httpHeaders = ImmutableMap.of();
+    private Map<String, String> httpHeaders = Map.of();
+    private File httpHeadersConfigFile;
+
+    @Config("http-event-listener.connect-http-headers.config-file")
+    @ConfigDescription("Path to a properties file containing custom HTTP headers, " +
+            "specified as key-value pairs (one per line, e.g., Header-Name=Header Value)")
+    public HttpEventListenerConfig setHttpHeadersConfigFile(File httpHeadersConfigFile)
+    {
+        this.httpHeadersConfigFile = httpHeadersConfigFile;
+        return this;
+    }
+
+    public Optional<@FileExists File> getHttpHeadersConfigFile()
```

Comment:
Put getter method first. Also, please put these methods under `httpHeaders` for consistency with fields.